packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -58,6 +58,7 @@ export const getComponentCss = (
         display: 'flex',
         position: 'relative',
         justifyContent: 'space-between',
+        alignItems: 'center',
```

Comment:
While this fixes the the alignment of the icons, isn't the different width still a problem?

<img width="335" alt="Bildschirmfoto 2023-12-06 um 09 10 20" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/e2098806-0c26-4985-a32e-9951863b6a0f">